### DISCUSSION TITLE
system.syslog Missing

### DISCUSSION THREAD

**Discussion by ShalvaGiorgadze** [Security-Onion-Solutions/securityonion#12795] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

40G

### Storage for /nsm

108G

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am new to SO, just installed it from official SO image to VMware environment. When examining dashboard, noticed a lot of system.syslog Missing events. Is this normal? How to fix this situation?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
If you look at one of the logs what do you see `event.category` field with any value?

**Reply by ShalvaGiorgadze** [reply]
I switched to the table view, if this is what you are asking and there a lot of logs with event.category *Missing
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/119173570/0539a67d-b3c8-4195-ae04-ed6f1b8f57d1)

**Reply by cm-ops** [reply]
Looks like some of your data is coming in and there is nothing in that field. Drill down into the logs to see what the traffic is. Look at the message field and see if there is a `category` field in the message.

**Reply by ShalvaGiorgadze** [reply]
Thank you for support. I have opened several messages, but there is no category field. This is the error.message filed, maybe this will be of help:
error.message: failed in Fleet agent final_pipeline: field [created] not present as part of path [event.created]
